### PR TITLE
added ttm thoth deployment and inference files

### LINKED ISSUE #521: Add Thoth TTM model deployment config and inference notebook
- [x] Add Seldon model deployment config for Thoth TTM model
- [x] Add inference notebook to interact with Thoth TTM model

### PR DESCRIPTION
**Pull request by oindrillac**
## Related Issues and Dependencies

closes #521
related https://github.com/open-services-group/community/issues/174 (1st acceptance criteria)

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

- In this PR, I added the seldon deployment config and inference notebook to interact with Thoth TTM model
- I also updated the path to where we are saving the test/train data to make it unique to repo (it was being overwritten by openshift origin)
- I also added the thoth PR data to the repo for enabling the local fetch option

## Next Steps

Integrate deployed TTM service with Thoth Bots

cc: @suppathak @aakankshaduggal

### CODE REVIEW

File: notebooks/time-to-merge-prediction/thoth-station/thoth-ttm-deployment-config.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+kind: SeldonDeployment
+metadata:
+  name: ttm-model-thoth
+spec:
+  name: ttm-model-thoth
+  predictors:
+  - componentSpecs:
+    - spec:
+        containers:
+        - image: quay.io/supathak/github-ttm-test:v2
```
Comment (suppathak) [inline notebooks/time-to-merge-prediction/thoth-station/thoth-ttm-deployment-config.yaml:11]: Hi @oindrillac ,
I believe in future we will be replacing this image to image from aicoe org right? or if it is ok to keep it ?

**Review by suppathak — COMMENTED** [review]
@oindrillac , just left a small comment.

File: notebooks/time-to-merge-prediction/thoth-station/thoth-ttm-deployment-config.yaml (hunk 1 above)
Comment (aakankshaduggal) [reply notebooks/time-to-merge-prediction/thoth-station/thoth-ttm-deployment-config.yaml:11]: @oindrillac - Should we merge this PR for now, or are we waiting on that issue to be resolved?

Comment (aakankshaduggal) [reply notebooks/time-to-merge-prediction/thoth-station/thoth-ttm-deployment-config.yaml:11]: Yeah that's alright. 🚀

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**